docs/contributing.rst

```diff
@@ -278,40 +278,48 @@ trailing underscore to distinguish them from the constants passes to
 Additional Functionality
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Transformers and Predictors
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Transformers
+^^^^^^^^^^^^
 
-A ``transformer`` object modifies the data it is given. An estimator may
-also be a transformer that learns the transformation parameters. The
-transformer object implements the ``transform`` method, i.e.
+A ``transformer`` object modifies the data it is given and by default outputs
+multiview data, a transformation of each input view. Additional parameters may
+allow for a single view output in the case of some algorithms. An estimator may
```

Comment:
maybe we should stick to a fix name. In the groupICA pr, it is `multiple_outputs`, but we can change it.

Reply:
Yes, I like defining a specific name. Is `multiview_output` better? Just worried that `multiple` is a bit vague. Could do `unified_output=False` or `single_output`as default. `multi_output` may even be better than `multiple`.

Reply:
My favorite is  `multiview_output`. I think this is the clearest name.

Reply:
oaky, I'll update the API here and should be reflected in groupICA. Thanks!